### ISSUE TITLE
KBFS not working

### ISSUE THREAD

**Issue by RyanSinger** [keybase/client#6244]
I get the popup warning.

my log id: 31c9c0c4ba1676ac249a2c1c

**Comment by strib** [comment]
Hi @RyanSinger -- can you please be more specific about what the "popup warning" says?

The KBFS log ends with the mysterious:

```
kbfsfuse error: (2) exit status 255
```

@gabriel: does the look familiar? anything interesting in the gui/updater logs?

**Comment by RyanSinger** [comment]
![image](https://cloud.githubusercontent.com/assets/2665386/23915318/f3d2a41c-08a5-11e7-8f4c-7837c828dcc5.png)

**Comment by strib** [comment]
Ah, got it.  We're intentionally killing KBFS because we're unable to mount it.  The error message seems self-explanatory -- is there something we can do to help you figure out your issue?

**Comment by RyanSinger** [comment]
The only listed service I have installed is Virtualbox, but that hasn't been launched since reboot. Do I have to uninstall and reboot to use KBFS?

**Comment by strib** [comment]
Hmm, not sure, sorry.  @gabriel?

**Comment by gabriel** [comment]
You can try some solutions from here:
https://github.com/keybase/client/wiki/Troubleshooting

**Comment by RyanSinger** [comment]
Uninstalled VirtualBox and rebooted and the problem is fixed. 

Feature request: Be able to have both installed :-(

**Comment by gabriel** [comment]
Yeah, hopefully Apple increases the number of device slots in future versions
